packages/components/README.md

```diff
@@ -6,56 +6,75 @@ Reusable UI components for Deriv.
 
 **In this document**
 
--   [Other documents](#other-documents)
 -   [Commands](#commands)
 -   [Storybook](#storybook)
 -   [Usage](#usage)
 
-## Other documents
-
--   [General](docs/README.md) - Contains general philosophy and overview of this package
--   [Modules docs](docs/Modules/README.md) - Contains implementation guides (i.e., scaffolding, code usage)
-
 ## Commands
 
 Setup:
+The `npm run bootstrap` command should be run in the root of the project, because it does not exist in the components package.js
 
 ```sh
-lerna bootstrap
+npm run bootstrap
 ```
 
+Build:
+The `npm run build` command is used to build the components package. It can be run both in the root directory (to build the entire project) and in the components directory (to build the components package)
+
+```sh
+npm run build
+```
+
+> The following commands are to be executed inside of the components directory and not the project root directory
```

Comment:
The following commands should executed inside of the components directory not in the root of the project.